centralized-sampling-tests/README.md

```diff
@@ -0,0 +1,34 @@
+# Centralized Sampling Integration Tests
+
+## Introduction
+
+This project is meant to be used to test that X-ray's centralized sampling strategies
+are working properly. This folder has sample apps configured for the integration tests
+under sample apps in languages that support centralized sampling. As of now the languages
+supported are Java and Go. To run these tests, first start up the collector, then
+start up the chosen sample app, then start running the tests.
+
+
+The design doc for this project is found here
+[Design Doc](https://quip-amazon.com/qcsdACbyatNi/X-Ray-Centralized-Sampling-Testing).
+The wiki for this project is found here
+[Wiki](https://w.amazon.com/bin/view/AWS/X-Ray/Development/SDK_and_Tools/Centralized-Sampling-Integration-Testing)
+
+## Run Locally
+
+### Set up collector
+To run locally first set up the collector with the correct configuration.
+It is possible that this is already done. The collector-contrib releases can be found here
+[Otel-Collector-Contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases).
```

Comment:
```suggestion
[Otel-Collector-Contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases).
```
Is there a reason this is using upstream collector contrib rather than the ADOT Collector? Also I do not believe this is the correct link to find upstream collector releases.

Reply:
That was the collector I have been using. No particular reason for using it over the ADOT collector, will change it. They both work the same for this.